ui/lib/config-ui/addon/components/messages/page/details.hbs

```diff
@@ -43,6 +43,16 @@
         (date-format (get @message attr.name) "MMM d, yyyy hh:mm aaa" withTimeZone=true)
       }}
     />
+  {{else if (eq attr.name "link")}}
+    {{#if (is-empty-value @message.link)}}
+      <InfoTableRow @label="Link" @value="None" />
+    {{else}}
```

Comment:
Could we remove this extra if/else and updated the block below to handle both cases?

```hbs
{{#each-in @message.link as |title href|}}
   <InfoTableRow @label="Link" @value={{or title "None"}}>
     {{#if @href}}
      <Hds::Link::Inline @icon="external-link" @href={{href}} data-test-link="message link">{{title}}</Hds::Link::Inline>
     {{/if}}
   </InfoTableRow>
{{/each-in}}